src/Microsoft.AspNetCore.DiagnosticsViewPage.Sources/project.json

```diff
@@ -0,0 +1,8 @@
+{
+  "version": "1.0.0-*",
+  "shared": "*.cs",
+  "dependencies": {},
+  "frameworks": {
```

Comment:
It doesn't seem to actually matter because it's a sources package. None of the code compiles in the context of this project. You could even have a compiler error and it doesn't care.

I just copied what some of the other shared source packages do.